index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;600;700&display=swap" rel="stylesheet">
+</head>
+<body>
+   <div class="page-wrapper">
+     <section class="head-top">
+       <div class="header-log">
+        <h2>My Logo</h2>
+       </div> 
+       <nav class="main-nav">
+        <a href="#">☰</a>
+        <ul class="nav-wrapper">
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
+       </nav>
+     </section> 
+     
+     <section class="headline">
+        <div class="intro-phrase">
+            <h1>I'm Frempong<br>Glad to see <br> you!</h1>
+        </div>
```

Comment:
Great job adding a section wrapping your introduction! This is perfect 😉.

Reply:
Thank You